nerfstudio/data/dataparsers/colmap_dataparser.py

```diff
@@ -434,7 +434,7 @@ def _downscale_images(self, paths, get_fname, downscale_factor: int, nearest_nei
                 path_out.parent.mkdir(parents=True, exist_ok=True)
                 ffmpeg_cmd = [
                     f'ffmpeg -y -noautorotate -i "{path}" ',
-                    f"-q:v 2 -vf scale=iw/{downscale_factor}:ih/{downscale_factor}{nn_flag} ",
+                    f"-q:v 2 -vf scale=round(iw/{downscale_factor}):round(ih/{downscale_factor}){nn_flag} ",
```

Comment:
ffmpeg seems doesn't support `round` function here. I am getting errors by running this command.

The bigger concern here is whether the rescaled image by ffmpeg has the exact same resolution as specified by `camera.rescale_output_resolution()`

I did some experiment locally and found that it could be quite tricky.